### DISCUSSION TITLE
Implementing demo for ML10:2023 Model Poisoning

### DISCUSSION THREAD

**Discussion by aryanxk02** [OWASP/www-project-machine-learning-security-top-10#112] [Machine Learning Security Topics]
Thinking of [ML10:2023 Model Poisoning](https://owasp.org/www-project-machine-learning-security-top-10/docs/ML10_2023-Model_Poisoning.html), we can create two scripts that, although carrying out the same operation (perhaps classification), which provide different outcomes. 

By this was, we can showcase model poisoning in action along with the theory corresponding to it.

Please share your ideas with me on this!

cc: @sagarbhure @shsingh @robvanderveer

**Comment by robvanderveer** [comment]
Model poisoning means directly manipulating the model (parameters). What scripts do you have in mind?

**Comment by sagarbhure** [comment]
In Federate Learining, we assume the attacker injects fake clients to a federated learning system and sends carefully crafted fake local model updates to the cloud server during training, such that the learnt global model has low accuracy for many indiscriminate test inputs. Towards this goal, our attack drags the global model towards an attacker-chosen base model that has low accuracy. example: OpenFL

Will put more ideas w.r.t other ML techniques and Model Poisoning Vector .
